backend/src/main/java/WebInterface.java

```diff
@@ -1,6 +1,21 @@
+import model.DigitalPart;
+import model.Order;
+import model.OrderedPart;
+
+import java.util.List;
+
 /**
  * Created by danie on 2017-09-28.
  */
 public interface WebInterface {
-    String getAllPrintsForStl(String stlID);
+    String dummy(String text);
+    //List<Order> getOrdersFromCustomer(int customerID);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)